src/ui-shell/sidenav/sidenav.component.ts

```diff
@@ -17,7 +17,7 @@ import { NavigationItem } from "../header/header-navigation-items.interface";
 @Component({
 	selector: "ibm-sidenav",
 	template: `
-		<nav class="bx--side-nav__navigation" role="navigation">
+		<nav class="bx--side-nav__navigation" role="navigation" [attr.aria-label]="ariaLabel">>
```

Comment:
```suggestion
		<nav class="bx--side-nav__navigation" role="navigation" [attr.aria-label]="ariaLabel">
```

Reply:
Also should probably be on multiple lines

Reply:
yes, thanks